### ISSUE TITLE
Detecting right click when mouse is moving

### ISSUE THREAD

**Issue by Slechani** [vasturiano/force-graph#346]
**Is your feature request related to a problem? Please describe.**
I have implemented contextual menus in my graph by using the onRightClick events, there are different menu options that are displayed based on what I click on. My problem is that my users are reporting that the contextual menus aren't working properly and I discovered that it happened because some of them were slightly moving their mouse when clicking.

The problem is that the onRightClickEvents won't trigger if i'm moving the mouse even very slightly so I don't think I have any way to fix this problem.

**Describe the solution you'd like**
I would like a way to make the right click events fire even when the mouse is moving as of right now just slightly moving the mouse a few pixels will prevent them from triggering

**Describe alternatives you've considered**

**Additional context**

**Comment by eunukasiko** [comment]
Yeah! The click events should be similar to `mousedown` rather than `click` events.
